modules/github_integration/config/locales/js-en.yml

```diff
@@ -47,3 +47,15 @@ en:
         empty: There are no pull requests linked yet. Link an existing PR by using the code <code>OP#%{wp_id}</code> in the PR description or create a new PR.
 
       github_actions: Actions
+
+      pull_requests:
+        message: Pull request %{pr_number} %{pr_link} for %{repository_link} has been %{pr_state} by %{github_user_link}.
+        referenced_message: >
+          %{github_user_link} referenced this work package in Pull request %{pr_number} %{pr_link} on %{repository_link}.
+        states:
+          opened: 'opened'
+          closed: 'closed'
+          draft: 'drafted'
+          merged: 'merged'
+          review_ready: 'marked ready for review'
```

Comment:
`review_ready` is wrong. It should be `ready_for_review`.

In `modules/github_integration/frontend/module/pull-request/pull-request-state.component.ts`:

```
export type PullRequestState = 'opened'|'closed'|'referenced'|'ready_for_review'|'merged'|'draft';
```

Currently, ready_for_review badge looks odd :) (and a little lifeless due to the color)

![image](https://user-images.githubusercontent.com/176055/230590057-7c822b89-d5a0-47ce-8d80-98b85d596cec.png)